App/SB/components/RadioButton/index.js

```diff
@@ -4,14 +4,18 @@ import { View, TouchableOpacity, Image } from 'react-native'
 import styles from './statics/styles'
 
 const RadioButton = props => {
-  const { selected, style, disabled } = props
+  const { selected, style, disabled, onPress } = props
 
   return (
-    <TouchableOpacity style={[styles.button, style, disabled && styles.disabled]}>
```

Comment:
This original `TouchableOpacity` had no function. It's use was actually causing the bug I understand you're trying to fix. This `TouchableOpacity` would "steal" the touch event from the `TouchableOpacity` in the parent component (the one that actually has the touch handler callback on it), and so if you touch the radio button, nothing happens.

The simple solution is to just change the use of `TouchableOpacity` here to a simple `View` since the `TouchableOpacity` contributed nothing anyway. This will allow the touch handler in the parent component to be triggered if you touch anywhere in the row, including the radio button.

Reply:
Ah, roger that! This is the first time I'm working with a react-native project, so I wasn't familiar with the distinction. I've reverted everything but the change in component type